src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -280,6 +278,17 @@ public final void setDefaultHeaders(final List<Header> headers) {
     }
 
     /**
+     * Print a curated set of important HTSJDK default settings to the logger
+     */
+    protected void printHTSJDKDefaults() {
+        logger.info("HTSJDK Defaults.COMPRESSION_LEVEL : " + Defaults.COMPRESSION_LEVEL);
+        logger.info("HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : " + Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS);
+        logger.info("HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : " + Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS);
+        logger.info("HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : " + Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE);
+        logger.info("HTSJDK Defaults.USE_CRAM_REF_DOWNLOAD : " + Defaults.USE_CRAM_REF_DOWNLOAD);
```

Comment:
Also, there is a VERBOSITY arg (LogLevel enum). You could keep the old behavior if VERBOSITY == Log.LogLevel.DEBUG.